### DISCUSSION TITLE
Open for collaboration (Germany)?

### DISCUSSION THREAD

**Discussion by anhaltintel** [baimard/gestion#371] [Q&A]
Hello there,

we're using your Nextcloud app for a while now in our Nextcloud version for German customers. We're planning on doing a bigger project for the German market next year which involves e-invoices especially for German companies.

We would like to collaborate with your project since we think it more advantegous to work together on an existing project since to start a completely new one.

Are you open to collaborate? We would like to localize the app for the German situation (units, legal and so on).

We would love to get your feedback. 

Kind regards

Chris, Andreas, and Team

**Comment by baimard** [comment]
Hello,

Yes I'm open of course.

We can plan a call and talk about your project ? Can we talk by mail ?

Benjamin
benjamin@cybercorp.fr

**Comment by anhaltintel** [comment]
Dear Benjamin,

thank you very much for your quick response. Of course, we can correspond
via e-mail.

Some background: Starting in 2025, German law changes for many companies
and they will have to issue e-invoices (so-called "E-Rechnungen" in
German).

In essence, what we want to do: We want to target small companies with an
invoicing tool integrated in our Nextcloud-based cloud platform called
CAMPUS. These companies will be able to generate their own
E-Rechnung-compliant invoices. Since you are already working on an
invoicing app in the Nextcloud environment, we would like to collaborate
with you in developing the German version of Gestion.

Looking forward to your reply

Kind regards

Christian


Danke und beste Grüße

Christian, Linda, Andreas & Co.
Team

<https://der-seminar.de/>
*DER SEMINAR <https://der-seminar.de/> - Tipps & Tools für Homeoffice und
digitale Arbeit*
Büro: Bernhardystr. 12, 06110 Halle (Saale)
<https://goo.gl/maps/irAFe6zkEG3mVgML6>
Telefon: +49 (0) 345 / 135 383 13 <+49034513538313>
E-Mail: ***@***.***

***@***.***>

<https://g.page/derseminar/review> <https://facebook.com/groups/derseminar>
<https://instagram.com/derseminar> <https://twitter.com/derseminar>
<https://onlinegeister.com/>
DER SEMINAR - Eine Marke der ATLAS Bildungskapital GmbH | UStID:
DE316871495 | Handelsregister: HRB 25275 | Registergericht: AG Stendal


Am Fr., 29. Dez. 2023 um 15:04 Uhr schrieb Benjamin <
***@***.***>:
